packages/wrangler/src/d1/execute.tsx

```diff
@@ -315,30 +320,23 @@ async function executeRemotely({
 	config,
 	name,
 	shouldPrompt,
-	batches,
-	json,
```

Comment:
we still want `json` mode though

Reply:
Fixed!

![image](https://github.com/cloudflare/workers-sdk/assets/23264/57e4fb49-433c-4d0c-a037-62ccf37c330b)